res/css/views/right_panel/_TimelineCard.scss

```diff
@@ -55,14 +55,19 @@ limitations under the License.
         margin-right: 0;
     }
 
-    .mx_EventTile:not([data-layout="bubble"]) .mx_EventTile_line {
-        padding-left: 36px;
-        padding-right: 36px;
-    }
+    .mx_EventTile:not([data-layout="bubble"]) {
+        .mx_EventTile_line {
+            padding-left: 36px;
```

Comment:
this would be a good opportunity to use spacing variables, but given a lot of this code is still in flux I'm hesitant to require it in this case.